ReactApp/.storybook/Wrapper.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+import { BrowserRouter } from "react-router-dom";
+import "typeface-roboto";
+import {
+  ThemeProvider,
+  StyledEngineProvider,
+  createTheme,
+} from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
+
+import AutomationStudioContext from "React-Automation-Studio/components/SystemComponents/AutomationStudioContext";
+
+import { io } from "socket.io-client";
+import lightTheme from "React-Automation-Studio/components/UI/Themes/lightTheme";
+import ReactVisCssBaseline from "React-Automation-Studio/components/SystemComponents/ReactVisCssBaseline";
+
+if (typeof window.socket === "undefined") {
```

Comment:
This is not a request, just out of curiosity: Would `window.socker === undefined` do the same trick?